### PR TITLE
Importing files

### PR DESCRIPTION
**Pull request by Ravishdeep10**
Added functionality to import objects from files. As of now, loading models with imported objects from scripts does not work unless you import the files again. Am looking into the best way to handle this other than just saving copies of the files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,9 @@ In order to run the gui:
 You may need to specify the gem5 root directory if the above yields import errors. On the command line:
 
     GEM5_HOME=<path to gem5>
+    
+## Importing objects from config scripts
+You can import custom made simobjects from python scripts into the gui. However before doing so you need to include the following code snippet in your script. 
+
+    import sys, os
+    sys.path.append(os.getenv('gem5_path'))
```
Comment (rohitdhamankar) [inline README.md:45]: is there a way to do this weithout having to add these lines? it would be a hassle to go through all .py files and add these lines

Comment (Ravishdeep10) [reply README.md]: I don't remember why I added those lines but you actually don't need to do that lol.

**Comment by rohitdhamankar** [comment]
> As of now, loading models with imported objects from scripts does not work unless you import the files again. Am looking into the best way to handle this other than just saving copies of the files.

can you save the path to the python file in the .ui file and load from there?

**Comment by Ravishdeep10** [comment]
I am looking into more dynamic solutions for this problem. There are a lot of edge cases where bugs can occur.

**Pull request merged**